pkg/security/apiserver/admission/sccadmission/admission.go

```diff
@@ -152,7 +152,7 @@ func (c *constraint) computeSecurityContext(a admission.Attributes, pod *coreapi
 	logProviders(pod, providers, errs)
 
 	if len(providers) == 0 {
```

Comment:
This condition is different from sccLister.List() is empty. Please give me a different message above.

Reply:
got it